### DISCUSSION TITLE
Transfer to another host

### DISCUSSION THREAD

**Discussion by walhallaRV** [microweber/microweber#971] [General]
Hi, i dint't find any information in the manual regarding change of machine / host and domainname (from playground to active). Any hint / solution to do this without losing anything? Thx & have a nive weekend ... cheers!

**Comment by phausleitner** [comment]
Haven't tried it yet, but only thing that comes to my mind - Modules - Backup - Create backup, and on the new site Upload your backup.. Really don't know, if it would work - maybe there will be needed changes in domainnames in DB.. but why not to try it :)

**Comment by markcjf** [comment]
Have transferred from server to localhost by downloading the microweber/project folder to local machine. Exported mysql DB and downloaded, created DB onto local mysql and imported .mysql file into DB. All working fine on localhost.
